### DISCUSSION TITLE
Passing relative path to a yaml file for parsing

### DISCUSSION THREAD

**Discussion by itsvamshiks** [mikefarah/yq#1398] [Q&A]
Hello. I have been using the library to convert a yaml file to json as follows

```
yq  -o=json filename.yaml"
```

Here, I navigate to the directory which has the file and using the command. Can I provide a relative path to the file as input instead (example below)

```
yq  -o=json ../apps/dev/filename.yaml"
```

**Comment by mikefarah** [comment]
Yes you can - though you have a double quotes at the end without a matching double quotes at the start...
